Automattic/wp-calypso
# Tag churn happychats with churn_chat_prompt

#### Proposed Changes

Demo video is in Testing instruction section.

* When user opens help center through the JITM CTA button, we will set a tag in help-center store
* Send `chat.tag` event to happychat server
* Add `chat.tag` event to happychat server: https://github.com/Automattic/happychat/pull/1860

#### Testing Instructions



https://user-images.githubusercontent.com/6586048/208105339-5c8e9c32-a8be-4a2d-ae5a-5d603034b68b.mp4

Setup:

- Sandbox: remember to also sandbox widgets.wp.com
- Sandbox: Make sure you are marked as a churn user, see instructions here: D93101-code
- Local happychat-hud: [Start happychat-hud](https://github.com/Automattic/happychat-hud/blob/staging/README.md#getting-started)
- Local happychat server: [Start happychat server and connect locally](https://github.com/Automattic/happychat/pull/1860)
- Local calypso: Checkout to this branch
- Local calypso: cd into `apps/happychat`
- Local calypso: run `yarn dev --sync`

Action:
- Happychat hud: Enter your local happychat hud and be available
- Go to http://calypso.localhost:3000/home/:site
- Calypso: Click on JITM to open up help center
- Calypso: Go to Live Chat and start a chat
- Happychat hud: check that the tag is showing on the top


#### Pre-merge Checklist





Related to https://github.com/Automattic/dotcom-forge/issues/1302
